src/main/java/com/loievroman/bookstoreapp/model/User.java

```diff
@@ -35,7 +44,50 @@ public class User {
 
     private String shippingAddress;
 
+    @ManyToMany(fetch = FetchType.EAGER)
```

Comment:
```suggestion
    @ManyToMany
```
Don’t  use FetchType.EAGER